glide.yaml

```diff
@@ -12,6 +12,8 @@ import:
   version: ^1
 - package: github.com/fatih/structtag
   version: ^0.1.0
+- package: go.uber.org/zap
+  version: master
```

Comment:
We had to do a bugfix in zap (see https://github.com/uber-go/zap/pull/614), and without it my tests would fail. I'll either have to pin to master or wait for the next release.

Reply:
Ah got it. We should see if we can cut a release before merging this in.